apps/prairielearn/src/pages/instructorAssessmentInstance/instructorAssessmentInstance.html.ts

```diff
@@ -180,15 +181,14 @@ export function InstructorAssessmentInstance({
                 <tr>
                   <th>Points</th>
                   <td colspan="2">
-                    ${renderEjs(import.meta.url, "<% include('../partials/pointsFormatter'); %>")}
-                    <span id="total-points"
-                      >${resLocals.assessment_instance.points.toString()}</span
-                    >
-                    <small
-                      >/<span id="total-max-points" class="text-muted"
-                        >${resLocals.assessment_instance.max_points.toString()}</span
-                      ></small
-                    >
+                    <span id="total-points">
+                      ${formatPoints(resLocals.assessment_instance.points)}
+                    </span>
+                    <small>
+                      /<span id="total-max-points" class="text-muted"
+                        >${formatPoints(resLocals.assessment_instance.max_points)}
```

Comment:
Presumably this is mushed onto one line to keep from introducing spaces where there shouldn't be any, is that right?

Reply:
Indeed. The score should stay with the slash that precedes it.